nexosim-util/src/helper_models.rs

```diff
@@ -0,0 +1,34 @@
+//! Helper models.
+//!
+//! This module contains helper models useful for simulation bench assembly.
+//!
+
+use std::time::Duration;
+
+use nexosim::model::{Context, InitializedModel, Model};
+
+/// A ticker model.
+///
+/// This model self-schedules with a provided period keeping simulation alive.
```

Comment:
/// This model self-schedules at the specified period, which can be used to keep the simulation alive.